### PR TITLE
Adding Database

### PR DESCRIPTION
**Pull request by Albertamoahjunior**
I have connected the backend to a database now.  
Instead of using json file as the database, now postgre is been used as database.

### CODE REVIEW

File: backend/dbconfig.ts
Comment (genes2160) [inline backend/dbconfig.ts:1]: This needs to be removed into env

File: backend/server.ts
Comment (genes2160) [inline backend/server.ts:1]: Why did you remove CORS ?

File: backend/src/controllers.ts
Comment (genes2160) [inline backend/src/controllers.ts:1]: Looks good

File: backend/dbconfig.ts
Comment (Albertamoahjunior) [reply backend/dbconfig.ts:1]: Yes please.

**Review by genes2160 — CHANGES_REQUESTED** [review]
image

This indicates that adding a new task does not work i.e. The entries never make it into the db

Always ensure that the entries make way into the DB and debug errors you find
image

File: backend/server.ts
Comment (Albertamoahjunior) [reply backend/server.ts:1]: I actually removed for testing purpose, to make sure the integration was working before I implement it again but I think I forgot to implement it again after I was done.

File: backend/src/controllers.ts
Comment (Albertamoahjunior) [reply backend/src/controllers.ts:1]: Yes please

**Comment by genes2160** [comment]
image
Is this how the UI is intended ????

File: frontend/src/components/taskcontainer.tsx
Comment (genes2160) [inline frontend/src/components/taskcontainer.tsx:1]: Next time delegate the 
```
http://localhost:2000
```
to its own file i.e. .env and import across so when the host changes there wont be a problem 

When it comes to this never hard code simply delegate to .env

Comment (Albertamoahjunior) [reply frontend/src/components/taskcontainer.tsx:1]: Yes please

**Review by genes2160 — APPROVED** [review]
Just few comments 
1. next time delegate a single .env that front and backend can make use of
i.e. structure can be 
|___.env (this ignored by git)
|____.env.example(this pushed to git)
|
|___frontend(folder)
|
|___backend(folder)


The both front and back can consume the .env

**Pull request merged**